mobify/platform-scaffold
# Handle Merlin's empty search submit

Handle Merlin's empty search submit

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-109

## Changes
- Add check for empty search to show error message instead of infinite loading results

## How to test-drive this PR
- Use merlin's connector
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Search for `     ` (some combination of spaces or an empty string). 
- Ensure that the notification error message pops up
